### PR TITLE
Prometheus/Loki: Run query explicitly instead of onblur in panel edit

### PR DESCRIPTION
**Pull request by torkelo**
* Run query explicitly instead of onblur in panel edit
* Also highlight the Run queries button when a change has been made 

[Edit-panel---prometheus-query-max---Dashboards---Grafana.webm](https://user-images.githubusercontent.com/10999/225337432-06aed543-dd72-416d-8a1b-09306ad0470b.webm)

### CODE REVIEW

**Comment by leeoniya** [comment]
maybe useful to add an Enter or Shift + Enter shortcut to run queries, too?

File: public/app/plugins/datasource/loki/components/LokiQueryField.tsx
Comment (matyax) [inline public/app/plugins/datasource/loki/components/LokiQueryField.tsx:79]: I think we can safely remove this prop and all related code, since it was only used for the on-blur run in dashboards.

**Review by svennergr — APPROVED** [review]
Loki change LGTM.

(Prom also, but I'll defer to the metrics squad).

> maybe useful to add an Enter or Shift + Enter or Ctrl + Enter shortcut to run queries, too?

@leeoniya yea, Shift+Enter is already working to run queries.

**Review by matyax — APPROVED** [review]

**Comment by torkelo** [comment]
@leeoniya I think they already exist

File: public/app/plugins/datasource/loki/components/LokiQueryField.tsx
Comment (torkelo) [reply public/app/plugins/datasource/loki/components/LokiQueryField.tsx]: Done removed this prop for both the prometheus and loki variant

**Review by matyax — APPROVED** [review]
🚀

**Comment by itsmylife** [comment]
Personally, I like the onBlurRunQuery maybe because I am used to it. And I see we add this in the changelog so users like me will be aware of that change 🤞. Highlighting the Run queries button is also a good touch. 
So PR LGTM but I wonder if we have any user complaints related to OnBlur behavior or any issues we encountered.

**Comment by torkelo** [comment]
> So PR LGTM but I wonder if we have any user complaints related to OnBlur behavior or any issues we encountered.

@itsmylife only from internal users so far, and some comments here https://github.com/grafana/grafana/issues/51979

**Pull request merged**
